install-guides/ubuntu20.md

```diff
@@ -0,0 +1,512 @@
+```
+php artisan config:cache
+```
+![image](https://user-images.githubusercontent.com/17537000/171812304-e00cc29b-453e-4bff-b398-ab453260bf57.png)
+
+* Laravel Horizon
+```
+php artisan horizon:install
+```
+```
+php artisan horizon:publish
+```
+![image](https://user-images.githubusercontent.com/17537000/171812367-970fea18-4150-46b2-9a3b-798cdaba464b.png)
+
+## Part 9.1 - Create an administrator account
+* Create a new admin user using php artisan
+```
+php artisan user:create
+```
+![image](https://user-images.githubusercontent.com/17537000/171835654-d6d42d7f-ca0d-43ee-a397-381a0c6d7533.png)
+
+## Part 10 - Return to the root account
+* Exit the `pixelfed` account and become root again.
+```
+exit
+```
+![image](https://user-images.githubusercontent.com/17537000/171812687-e34fd89f-6bd0-4724-aed6-ef60a7dfbdc1.png)
+
+## Part 11 - Nginx and Certbot - Install
+* Install stock nginx, Install certbot to handle the TLS certificate and enable the engine service to autostart.
+```
+apt -y install nginx certbot python3-certbot-nginx
+```
+```
+systemctl enable nginx
+```
+![image](https://user-images.githubusercontent.com/17537000/171813872-b3b2066c-fa90-4e4d-98bb-8ca3f797ffed.png)
+
+* Check the certbot timer is running (Certbot will renew TLS certs automatically)
+```
+systemctl status certbot.timer
+```
+![image](https://user-images.githubusercontent.com/17537000/171818819-231ada60-060d-4eb0-8692-e9cea151307b.png)
+
+
+## Part 11.1 - Configure certbot
+```
+rm /etc/nginx/sites-enabled/default
+```
+* Generate a new TLS Certificate for `pixelfed.au`.
+```
+certbot -d pixelfed.au
+```
+![image](https://user-images.githubusercontent.com/17537000/171825178-c453a2db-ad03-4ba8-8536-b794898c272a.png)
+
+## Part 11.2 - Configure nginx
+* Copy the nginx pixelfed.conf
+```
+cp /home/pixelfed/pixelfed/contrib/nginx.conf /etc/nginx/sites-available/pixelfed.conf
+```
+
+* Open the nginx pixelfed.conf file
+```
+nano /etc/nginx/sites-available/pixelfed.conf
+```
+
+* Edit these lines to match your new instance
+
+> * server_name has to be changed twice
+> * root needs to be changed to the correct path
+> * fastcgi_pass needs to be updated to match the fpm conf
+> * ssl_certificate and ssl_certificate_key need to be updated with the certbot path
+
+```
+    server_name pixelfed.au;
+    root /home/pixelfed/pixelfed/public;
+        
+    ssl_certificate /etc/letsencrypt/live/pixelfed.au/fullchain.pem;       # generate your own
+    ssl_certificate_key /etc/letsencrypt/live/pixelfed.au/privkey.pem;   # or use letsencrypt
+
+    fastcgi_pass unix:/run/php/php7.4-fpm-pixelfed.sock;
+```
+![image](https://user-images.githubusercontent.com/17537000/171829099-3541b28c-0953-471a-8385-a36fd91c1606.png)
+
+* Add a symlink to the nginx sites-enabled folder
+```
+ln -s /etc/nginx/sites-available/pixelfed.conf /etc/nginx/sites-enabled/
+```
+
+* Reload nginx to load the new enabled site.
+```
+systemctl reload nginx
```

Comment:
do you need to enable/start nginx too? i didn't see that earlier

Reply:
reload is different. it updates the config after editing the pixelfed.conf

Reply:
i know what reload does, i'm asking if you also need to enable/start nginx so that it's actually running